backend/app/routes/scraping.py

```
@@ -160,6 +161,45 @@ async def quick_scrape_croatia(
+            return await scrape_infozagreb_events(max_pages=request.max_pages)
+
+        background_tasks.add_task(run_info_scrape)
+
+        return ScrapeResponse(
+            status="accepted",
+            message=f"InfoZagreb scraping task started in background for {request.max_pages} pages",
+            task_id=task_id,
+        )
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Failed to start InfoZagreb scraping: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)